rules/net/dns-servers.yara

```diff
@@ -1,4 +1,4 @@
-rule go_dns_refs {
+rule go_dns_refs_local {
```

Comment:
There were two `go_dns_refs` rules and there was a chance one would clobber the other:
- https://github.com/chainguard-dev/malcontent/blob/0acb2e0e98915c8fee1d6723dfbf1a29d861ed5c/rules/net/dns.yara#L1-L11
- https://github.com/chainguard-dev/malcontent/blob/0acb2e0e98915c8fee1d6723dfbf1a29d861ed5c/rules/net/dns-servers.yara#L1-L10